src/components/PeopleFilter/PeopleFilters.tsx

```diff
@@ -0,0 +1,133 @@
+import classNames from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from '../SearchLink';
+import { SearchParams, getSearchWith } from '../../utils/searchHelper';
+
+enum ESexFilter {
+  Male = 'm',
+  Female = 'f',
+}
+
+const centuries: string[] = ['16', '17', '18', '19', '20'];
+
+export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const sexParam = searchParams.get('sex');
+  const queryParam = searchParams.get('query') ?? '';
+  const centuriesParam = searchParams.getAll('centuries') ?? [];
+
+  const getGenderLinkStyle = (gender: string | null) => (
+    classNames({ 'is-active': gender === sexParam })
+  );
+
+  const getCenturiesLinkStyle = (centery: string) => (
+    classNames('button', 'mr-1', {
+      'is-info': centuriesParam.includes(centery),
+    })
+  );
```

Comment:
It is better to define this functions outside the component so that it is not created every time a new render is performed.